packages/cactus-plugin-keychain-aws-sm/src/test/typescript/integration/plugin-keychain-aws-sm.test.ts

```diff
@@ -0,0 +1,124 @@
+import test, { Test } from "tape-promise/tape";
+import { v4 as internalIpV4 } from "internal-ip";
+
+import {
+  Containers,
+  LocalStack,
+  K_DEFAULT_LOCALSTACK_HTTP_PORT,
+} from "@hyperledger/cactus-test-tooling";
+
+import { v4 as uuidv4 } from "uuid";
+
+import { LogLevelDesc } from "@hyperledger/cactus-common";
+
+import {
+  IPluginKeychainAwsSmOptions,
+  PluginKeychainAwsSm,
+  AwsCredentialType,
+} from "../../../main/typescript/public-api";
+
+const logLevel: LogLevelDesc = "TRACE";
+
+test("get,set,has,delete alters state as expected", async (t: Test) => {
+  const localStackContainer = new LocalStack({});
```

Comment:
@jagpreetsinghsasan Recommending to pass in the `logLevel` to the LocalStackContainer options instead of just the empty object.